optimum/exporters/openvino/model_configs.py

```diff
@@ -74,6 +76,8 @@
 
 
 def init_model_configs():
+    TasksManager._LIBRARY_TO_SUPPORTED_MODEL_TYPES["open_clip"] = {}
```

Comment:
shouldn't overwrite if exist (could be the case for other export than openvino)
```suggestion
    if "open_clip" not in TasksManager._LIBRARY_TO_SUPPORTED_MODEL_TYPES:
        TasksManager._LIBRARY_TO_SUPPORTED_MODEL_TYPES["open_clip"] = {}
```